### PR TITLE
Nick m updates

### PR DESCRIPTION
**Pull request by Arkanisbot**
get_panel function searches for the appellate judges that may be assigned to the documents. The list was originally pulled from wikipedia, but seems to be partially updated (35 on panel_members compared to 29 active judges). This does not include state level immigration judges, therefore any case that get panel returns a list on is an appellate case. All 170 manually scraped cases are appellate cases. 

Use the get_panel function to determine if appellate and sets the class variable self.appellate to True or False. This is used by get_city and get_state, it can also be used by other functions in future designs.

Get_state searches for the keyword file or files, which are listed on every appellate case in the sentence including the initial court location. It then checks each token in the sentence to match up to the list of abbreviations in the abbrev dictionary. Once it has a match it sets a class variable to that abbreviation. Future changes could set the state to the full name and then reference the abbreviation dictionary when using get_city. If not an appellate case this function searches for the first abbreviation that's in the dictionary and uses that. This could be a potential concern if a non-appellate case has a pair of letters that appear in the document before the state. This could be alleviated by creating a matcher pattern for each state abbreviation and then searching the document for those. 

get_city uses the state abbreviation to search the court_locations json and return a citycache set that houses the potential cities from a document. Then the function does the same search as get_state, but instead of returning the abbreviation it returns the city listed.

### CODE REVIEW

**Review by FrankBellamy — CHANGES_REQUESTED** [review]
Before I approve this:
(1) I would want some coherence / resolution between you and Jacob on get_panel
(2) I do not want to separate out fields into a separate file at this late date, especially given that the code you are using for fields seems to be based on an outdated version of the code.

File: app/ocr.py
Comment (PatrickRaborn) [inline app/ocr.py:291]: missing attributes

File: app/ocr.py
Comment (PatrickRaborn) [inline app/ocr.py:154]: needed for get date inside class

File: app/ocr.py
Comment (PatrickRaborn) [inline app/ocr.py:566]: implement missing attributes

**Review by PatrickRaborn — CHANGES_REQUESTED** [review]
Review comments to implement missing lines of code before approval can be made

**Pull request closed without merging**
